src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_diagnostics_extension_install.yaml

```diff
@@ -605,7 +605,7 @@ interactions:
       User-Agent:
       - AZURECLI/2.19.1 azsdk-python-azure-mgmt-storage/17.0.0 Python/3.7.4 (Windows-10-10.0.19041-SP0)
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/cli_test_vm_vmss_diagnostics_extension000001/providers/Microsoft.Storage/storageAccounts?api-version=2021-01-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/cli_test_vm_vmss_diagnostics_extension000001/providers/Microsoft.Storage/storageAccounts?api-version=2021-02-01
```

Comment:
Please make sure it can pass, or mark it if it fails. I do not recommend editing recording file. Just run it. I noticed some are rerun while some are manually edited. Why?

Reply:
It failed in some vm logic which has nothing to do with storage bump when I rerun locally. Maybe you could help detect the real reason?